api/build.gradle

```diff
@@ -177,18 +186,40 @@ dependencies {
     implementation group: 'org.springframework.cloud', name: 'spring-cloud-starter-netflix-hystrix', version: '2.2.7.RELEASE'
     implementation group: 'org.springframework.cloud', name: 'spring-cloud-starter-bootstrap', version: '3.0.1'
 
+    implementation group: 'uk.gov.service.notify', name: 'notifications-java-client', version: '3.17.0-RELEASE'
+
+    implementation group: 'uk.gov.hmcts.reform', name: 'logging', version: versions.reformsJavaLogging
+    implementation group: 'uk.gov.hmcts.reform', name: 'logging-appinsights', version: versions.reformsJavaLogging
+    implementation group: 'uk.gov.hmcts.reform', name: 'logging-httpcomponents', version: versions.reformsJavaLogging
+    implementation group: 'uk.gov.hmcts.reform', name: 'logging-spring', version: versions.reformsJavaLogging
+    implementation 'junit:junit:4.12'
```

Comment:
aren't we using junit5?

Reply:
If we need vintage Junit 4 features we can use this dependency...
testImplementation group: 'org.junit.vintage', name: 'junit-vintage-engine', version: '5.7.1'

Reply:
I'd be interested to know if we did. I've not hit any issues with junit 5 before

Reply:
for now will use junit5 but if we need vintage later we can add that